app/views/positions/edit.haml

```diff
@@ -4,3 +4,19 @@
 
 = button_to "Remove #{@position.name_and_department}",
   @position, method: :delete
+%p Subscribe to email notifications about this position?
```

Comment:
This text is a little awkward, and I'd like to find a way to visually organize this page a little bit more.